libp2p.go

```diff
@@ -37,8 +37,7 @@ func ChainOptions(opts ...Option) Option {
 // transport protocols;
 //
 // - If no multiplexer configuration is provided, the node is configured by
-// default to use the "yamux/1.0.0" and "mplux/6.7.0" stream connection
-// multiplexers;
+// default to use the "yamux/1.0.0" stream connection multiplexer;
```

Comment:
I left this as it was before but should this be either just the word `yamux` or `"/yamux/1.0.0"`? If so any preference? Maintainers can feel free to merge without this change, or make any change they see fit here.